src/frontend/src/pages/InfoPage.tsx

```diff
@@ -22,13 +22,17 @@ const InfoPage: React.FC = () => {
         </Box>
         <Grid container spacing={2}>
           <Grid item md={4} lg={3}>
-            <ExternalLink
-              icon={faScroll}
-              description={'Glossary Document'}
-              link={'https://docs.google.com/document/d/1_kr7PQxjYKvBTmZc8cxeSv5xx0lE88v0wVXkVg3Mez8/edit?usp=sharing'}
-            />
+            <Typography>
+              <ExternalLink
+                icon={faScroll}
+                description={'Glossary Document'}
+                link={'https://docs.google.com/document/d/1_kr7PQxjYKvBTmZc8cxeSv5xx0lE88v0wVXkVg3Mez8/edit?usp=sharing'}
+              />
+            </Typography>
```

Comment:
we should follow principles of abstraction and put the typography in the external link component where it needs to be, rather than wrapping every usage of the external link component